Detectors/TPC/base/include/TPCBase/ParameterGEM.h

```diff
@@ -54,10 +59,11 @@ struct ParameterGEM : public o2::conf::ConfigurableParamHelper<ParameterGEM> {
   float AbsoluteGain[4] = {14.f, 8.f, 53.f, 240.f};              ///< Absolute gain
   float CollectionEfficiency[4] = {1.f, 0.2f, 0.25f, 1.f};       ///< Collection efficiency
   float ExtractionEfficiency[4] = {0.65f, 0.55f, 0.12f, 0.6f};   ///< Extraction efficiency
-  float TotalGainStack = 1644.f;                                 ///< Total gain of the stack for the EffectiveMode
-  float KappaStack = 1.2295f;                                    ///< Variable steering the energy resolution of the full stack for the EffectiveMode
-  float EfficiencyStack = 0.473805f;                             ///< Variable steering the single electron efficiency of the full stack for the EffectiveMode
+  float TotalGainStack = 2000.f;                                 ///< Total gain of the stack for the EffectiveMode
+  float KappaStack = 1.205f;                                     ///< Variable steering the energy resolution of the full stack for the EffectiveMode
+  float EfficiencyStack = 0.528f;                                ///< Variable steering the single electron efficiency of the full stack for the EffectiveMode
   AmplificationMode AmplMode = AmplificationMode::EffectiveMode; ///< Amplification mode [FullMode / EffectiveMode]
+  PadResponseMode PRFmode = PadResponseMode::FullPadResponse;    ///< Treatment of the pad response [FullPadResponse / ProjectivePadResponse]
```

Comment:
In general I think the projective PRF is more than enough and should be the default. Nevertheless I think it would be nice to have at least the option to test with PRF enabled. In particular for very short drifts it also makes an appreciable difference in the dE/dx and the charge distributions so it would be really good to have this